### PR TITLE
[YUNIKORN-348] update readme for release update

### PR DESCRIPTION
**Pull request by wilfred-s**
Describe what needs to be updated and which content is kept in which
location.
Focussed on new documentation version linked to a release.

### CODE REVIEW

**Comment by yangwwei** [comment]
hi @wilfred-s  thanks for updating the doc.
@TaoYang526 , pls help to review this one, as you just have gone through all the steps : )
thanks

File: README.md [hunk 1]
```diff
@@ -63,42 +61,121 @@ yarn run start
...
+### Update the download page
+The download page contains the link to the Apache source download, signatures etc.
+The table on the page has a specific layout that is correctly rendered after going through the docusaurus build.
+Please do not change the release entry layout in the table.
+
+First step is to update the `latest release` line to the correct version.  
+Second step is to update the table: copy an existing entry and change the links and details to point to the new release.
+All links must be updated:
+* 3 for the download (source tar, checksum and signature)
+* 2 for the docker images (scheduler and admission controller)
```
Comment (TaoYang526) [inline README.md:166]: Here should have 3 including the web image.

**Comment by TaoYang526** [comment]
I have looked through the updates, all of them LGTM, only one minor problem commented above.
Thanks @wilfred-s for improving this readme doc, which can help a lot to better understand how the website works and how to release a new version.

File: README.md (hunk 1 above)
Comment (wilfred-s) [reply README.md]: fixed

**Comment by wilfred-s** [comment]
committing based on the previous LGTM from Tao

**Pull request closed without merging**